### ISSUE TITLE
save to owncloud 

### ISSUE THREAD

**Issue by manesh1** [RainLoop/rainloop-webmail#1459]
**RainLoop version 1.10.0.5.192, browser, OS: centos 

**Expected behavior and actual behavior: want the feature to save the attachment in owncloud server

**Comment by akmet** [comment]
duplicate of https://github.com/RainLoop/rainloop-webmail/issues/1329

**Comment by DJCrashdummy** [comment]
@akmet no, it is not! #1329 is about "**sending** attachments" directly from own-/nextcloud.

and "**save** to owncloud" is already available: https://github.com/RainLoop/rainloop-webmail/issues/1329#issuecomment-287712191

**Comment by akmet** [comment]
@DJCrashdummy yeah, you are right but I linked the other thread so others could have a look at it.
I'm sorry that I didn't made it clear from the beginning and linked the comment instead of the whole thread.

**Comment by DJCrashdummy** [comment]
@RainLoop i guess this can/should be closed since it is available: https://github.com/RainLoop/rainloop-webmail/issues/1329#issuecomment-287712191
